src/lighteval/metrics/normalizations.py

```diff
@@ -444,15 +445,16 @@ def remove_punc(text: str) -> str:
     return "".join(ch for ch in text if ch not in PUNCT)
 
 
+@requires(Extras.MULTILINGUAL)
 def get_multilingual_normalizer(lang: Language, lower: bool = True) -> Callable[[str], str]:
     """Get a normalizer function for the specified language.
 
     Returns:
         Callable[[str], str]: A function that normalizes text for the specified language
     """
-    tokenizer = get_word_tokenizer(lang)
 
     def _inner_normalizer(text: str) -> str:
+        tokenizer = get_word_tokenizer(lang)
```

Comment:
This moves has it lazy init; it doesn't change anything except the scope as `get_word_tokenizer` is LRU cached